ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -75,6 +75,25 @@ vlan_interfaces:
         vrf: {{ svi_ip_helpers[helper_ip].source_vrf | arista.avd.default }}
 {%                 endfor %}
 {%             endif %}
+{%             if svi.ospf.enabled is arista.avd.defined(true) %}
+    ospf_area: {{ svi.ospf.area | arista.avd.default(0) }}
+    ospf_network_point_to_point: {{ svi.ospf.point_to_point | arista.avd.default(false) }}
+{%             endif %}
+{%             if svi.ospf.cost is arista.avd.defined() %}
```

Comment:
remove empty `()` (same elsewhere) - i know it is inconsistent everywhere, but let's try to remove it with we build new stuff.

Reply:
Fixed in all network_services templates.